cockpit/test/courses.yaml

```diff
@@ -0,0 +1,144 @@
+openapi: "3.0.0"
```

Comment:
I think there is no need to have that test folder at all, right?

Reply:
I need it for the app-connector and to view the specs when clicking on the api

Reply:
removed